src/Controller/Project/ProcesVerbalController.php

```diff
@@ -46,14 +48,26 @@ public function add(
             throw new AccessDeniedException('Cette étude est confidentielle');
         }
 
-        $proces = new ProcesVerbal();
-        $etude->addPvi($proces);
+        if ((null == $etude->getPvr() && 'pvr' == $type) || 'pvi' == $type) {
```

Comment:
```suggestion
        if ((null === $etude->getPvr() && 'pvr' === $type) || 'pvi' === $type) {
```

Triple égal pour vérifier les types en plus de la valeur.

Reply:
pourquoi faut-il supprimer les lignes 49 et 50 ici ?

Reply:
Ca doit être l'UI de Github qui t'induit en erreur. Les lignes 49 & 50 sont supprimés dans ta PR.

De mon coté, mon commentaire porte sur la ligne 51, avec la suggestion de remplacer `==` par `===`

Reply:
Ah oui, c'est fait alors :) Merci !
Désolé pour l'incompréhension